examples/asr/speech_to_label.py

```diff
@@ -47,10 +47,10 @@
 # Task 2: Voice Activity Detection
 
 ## Preparing the dataset
-Use the `process_vad_data.py` script under <NEMO_ROOT>/scripts in order to prepare the dataset.
+Use the `get_speech_commands_data.py` script under <NEMO_ROOT>/scripts in order to prepare the dataset.
```

Comment:
Vad data is not necessarily speech command data. @fayejf is this fine?

Reply:
no this is not fine. _process_vad_data_ also include others light rebalance/preprare demo data for notebook, it should be _process_vad_data_ here. And I can't find the script under scripts, could you bring it back? @VahidooX

Reply:
renamed it back to process_vad_data, but it looks like that it always download this dataset: "google speech command v2", yes?
And I updated the notebooks. Would you please check that it works like this now?

Reply:
yes. It includes downloading the google speech command v2 but also a lot of other stuff. we could further clean it up (say extract downloading part and reuse it in different tasks) but let's do it out of RC? 

Yes please see below.